bin/install

```diff
@@ -17,6 +17,9 @@ function get_arch_short {
     "x86_64")
       local arch_short="x64"
       ;;
+    "arm64")
```

Comment:
I would suggest merging this branch with the upper one to keep things consistent, i.e. have `"x86_64" | "arm64"` in line 17.